src/containers/lock/containers/LockForm/styles.js

```diff
@@ -1,6 +1,7 @@
 import styled from "styled-components";
 
 import theme from "styles/theme";
+import { media } from "../../../../styles/media";
```

Comment:
Please use the `styles/media` instead of the relative path.

Reply:
hm strange i certainly didn't do this, but fixed it anyway.